test/data/create_corner_cases_zim_file

```diff
@@ -1,7 +1,24 @@
 #!/usr/bin/env bash
 
 cd "$(dirname "$0")"
-rm -f corner_cases.zim
+
+# The following symbols (that would be nice to include in testing) are not
+# allowed under NTFS and/or FAT32 filesystems, and would result in the
+# impossibility to git clone (or rather checkout) the libkiwix repository under
+# Windows:
+#
+#  ?
+#  =
+#  + (that's a pity, since the + symbol in a ZIM filename is replaced with the
+#     text 'plus' when the ZIM file is added to kiwix-serve's library and it
+#     would be nice to test that functionality)
+#
+# Assuming that tests are NOT run under Windows, above symbols can be included
+# in testing if the file is renamed while copying to the build directory (see
+# test/meson.build), though that would make maintenance slightly more confusing.
```

Comment:
We will compile libkiwix on windows and it would be a good thing to test it also.

One thing will could do is rename the file only on linux and don't rename it on windows.
(And adapt the code accordingly). But in this case, that would make maintenance more than slightly more confusing.

Reply:
Should we do anything about it now/in this PR?

Reply:
No, we can leave it at it is.